### ISSUE TITLE
Unknown config option "viewType"

### ISSUE THREAD

**Issue by Frolanta** [filestack/filestack-react#66]
Hi,

The picker option "viewType" ins't available.
https://filestack.github.io/filestack-js/interfaces/pickeroptions.html#viewtype

```
<ReactFilestack
  apikey='FS_API_KEY'
  options={{
    viewType: 'grid',
     maxSize: 10 * 1024 * 1024,
   }}
   preload={true}
/>
```
The above code give : `Uncaught (in promise) Error: Unknown config option "viewType"`

version: 2.0.6

**Comment by AndrzejSala** [comment]
Hi @Frolanta 
A few days ago we deployed a new 3.0.0 version of filestack-js and 'viewType' is the newly added option.
Filestack-react 2.0.6 is still using filestack-js 2.1.0 under the hood, so it dosn’t recognize ‘viewType’ option.

We are going to release a new version of filestack-react at the start of next week.

**Comment by Frolanta** [comment]
Thanks for your answer. I'm looking forward for the new version then 🙂

**Comment by AndrzejSala** [comment]
@Frolanta 
We just released 3.0.0 and this option should be included.
